README.md

```diff
@@ -16,10 +16,3 @@ Talk to us on our [mailing list](https://groups.google.com/forum/#!forum/nusskna
 ## License
 
 **Nussknacker** is published under [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0).
-
-
-## Scala compatibility
```

Comment:
What about leaving this with notice that we support both 2.11.x and 2.12.x? Because there will be still some not supported versions...

Reply:
true, I'll add it back